desktop_version/src/Editor.cpp

```diff
@@ -1459,15 +1461,18 @@ void editorrender(void)
                 toolname = "???";
                 break;
             }
-            int toolnamelen = graphics.len(toolname);
-            graphics.fill_rect(0,197,toolnamelen+8,11, graphics.getRGB(32,32,32));
-            graphics.fill_rect(0,198,toolnamelen+7,10, graphics.getRGB(0,0,0));
-            graphics.bprint(2,199, toolname, 196, 196, 255 - help.glow);
-
-            graphics.fill_rect(260,197,80,11, graphics.getRGB(32,32,32));
-            graphics.fill_rect(261,198,80,10, graphics.getRGB(0,0,0));
-            font::print(PR_BOR | PR_CJK_HIGH | PR_RIGHT, 316, 199, "("+help.String(ed.levx+1)+","+help.String(ed.levy+1)+")", 196, 196, 255 - help.glow);
-
+            int bgheight = 2 + font::height(0);
+            int toolnamelen = font::len(0, toolname);
+            graphics.fill_rect(0,207-bgheight,toolnamelen+8,bgheight+1, graphics.getRGB(32,32,32));
+            graphics.fill_rect(0,208-bgheight,toolnamelen+7,bgheight, graphics.getRGB(0,0,0));
+            font::print(PR_BOR | PR_CJK_HIGH, 2,199, toolname, 196, 196, 255 - help.glow);
+
+            char coords[8];
+            vformat_buf(coords, sizeof(coords), "({x},{y})", "x:int, y:int", ed.levx+1, ed.levy+1);
```

Comment:
Unless the style of coordinates is going to be localizable, this should just be using `SDL_snprintf`.

Reply:
Hm, I've been thinking it'd actually be good to move to consistently using one style of string formatting instead of two ones mixed together?

Reply:
I'm not too sure. Ideally, if the string is known at compile-time, then we should use `SDL_snprintf`, because it can be easily checked for correctness by compilers and linters (it's annotated with `SDL_PRINTF_FORMAT_STRING` and `SDL_PRINTF_VARARG_FUNC`) without needing to do run-time testing. VFormat makes sense where it's used currently because it was designed for strings not known at compile-time, that may change at run-time, and those strings are untrusted.

Reply:
Fair enough.

It might actually be that we'll end up being asked to make this translatable (because `(1;2)` happened to develop as a strong convention in a specific language or something) but I guess we'll see, and we can always change it back in that case.